mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/param/DefaultEncoding.java

```diff
@@ -15,9 +15,17 @@
 import java.lang.annotation.Target;
 
 /**
- * Marks that if there were no value provided for the annotated element (which must be of type {@link String}), Mule's default
- * encoding should be used. This annotation also implies the optionality of the annotated element (just as if it were annotated with
- * {@link Optional}).
+ * This annotation is meant to be applied on fields of classes
+ * or on operation's arguments.
+ * <p>
+ * This annotation signals that the target field should be injected
+ * with Mule's default encoding.
+ * <p>
+ * This implies the following restrictions:
+ * <p>
+ * <lu>
```

Comment:
it's `<ul>`, not `<lu>`, and it doesn't make sense to make a table for just one restriction